exporter/datadogexporter/config/config.go

```diff
@@ -104,6 +104,14 @@ type TracesConfig struct {
 	// all entries must be surrounded by double quotes and separated by commas.
 	// ignore_resources: ["(GET|POST) /healthcheck"]
 	IgnoreResources []string `mapstructure:"ignore_resources"`
+
+	// SpanNameRemappings is the list of datadog span names and preferred names to map to. This can be used to
+	// automatically map Datadog Span Operation Names to an updated value, and is useful when a user wants to
+	// shorten or modify span names to something more user friendly in the case of instrumentation libraries with
+	// particularly verbose names. All entries must be surrounded by double quotes and be separated by commas, with each entry
+	// between old and updated name separated by a space.
```

Comment:
Why don't we just use a `map[string]string` for this instead of having to parse it ourselves?

Reply:
yea i guess i was thinking about how it would apply to an env var but we can worry about that if we ever choose to go that route, will update

Reply:
updated to a map